package.json

```diff
@@ -105,15 +106,16 @@
     "karma-jasmine-html-reporter": "0.2.2",
     "karma-junit-reporter": "1.1.0",
     "karma-requirejs": "0.2.6",
-    "karma-selenium-webdriver-launcher": "0.0.4",
+    "karma-selenium-webdriver-launcher": "github:brian-smith-tcril/karma-selenium-webdriver-launcher#loosen-deps",
     "karma-sourcemap-loader": "0.3.7",
     "karma-spec-reporter": "0.0.20",
-    "karma-webpack": "2.0.9",
+    "karma-webpack": "^5.0.1",
     "plato": "1.7.0",
     "react-test-renderer": "16.4.0",
     "selenium-webdriver": "3.4.0",
     "sinon": "2.3.5",
     "squirejs": "0.1.0",
-    "stylelint-formatter-pretty": "1.0.3"
+    "stylelint-formatter-pretty": "1.0.3",
+    "webpack-cli": "^5.1.4"
```

Comment:
Do we intend to keep this?  If not, we should probably add an explicit temp comment.

(I don't have any objections to keeping it, btw.)

Reply:
We use the `webpack` cli in CI. In webpack 2 the cli was included in the `webpack` package, now it has been moved out into the `webpack-cli` package. Unless we plan on installing `webpack-cli` some other way I think it makes sense to keep it as a dev dependency.